src/screens/FeedScreen/Content.js

```diff
@@ -329,9 +342,10 @@ Content.propTypes = {
 export default Content;
 
 export const styles = StyleSheet.create({
-  containerImage: {
-    flex: 1
-  },
+  containerImage: (isBlurred) => ({
+    flex: 1,
+    height: isBlurred ? dimen.normalizeDimen(422) : 100
+  }),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The style for the image container now depends on whether the post is blurred. This should help adjust the layout for blurred posts, but make sure this doesn't cause any visual issues or overlaps between components.
```diff
-  containerImage: {
-    flex: 1
-  },
+  containerImage: (isBlurred) => ({
+    flex: 1,
+    height: isBlurred ? dimen.normalizeDimen(422) : 100
+  }),
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->